Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp

```diff
@@ -872,6 +877,26 @@ void MediaPlayerPrivateGStreamerMSE::emitPlayReadySession(PlayreadySession* sess
 #endif
 #endif
 
+#if (ENABLE(LEGACY_ENCRYPTED_MEDIA) || ENABLE(LEGACY_ENCRYPTED_MEDIA_V1)) && USE(OCDM)
+void MediaPlayerPrivateGStreamerMSE::emitOpenCDMSession()
+{
+    if (!m_cdmSession)
+        return;
+
+    CDMSessionOpenCDM* cdmSession = getOpenCDMSession();
+    const String& sessionId = cdmSession->sessionId();
+    if (sessionId.isEmpty())
+        return;
+
+    for (auto it : m_appendPipelinesMap) {
+            gst_element_send_event(it.value->pipeline(), gst_event_new_custom(GST_EVENT_CUSTOM_DOWNSTREAM_OOB,
+                gst_structure_new("drm-session", "session", G_TYPE_STRING, sessionId.utf8().data(), nullptr)));
+            it.value->setAppendState(AppendPipeline::AppendState::Ongoing);
+    }
+    GST_DEBUG("emitted OCDM session on pipeline");
```

Comment:
Call is OpenCDM. I refactored this on V3 to avoid abbreviating to OCDM. I am not going to force changing OCDM to OpenCDM everywhere but this place does not need to keep that outdated naming.